### ISSUE TITLE
Exception in `parse_guild_member_add`

### ISSUE THREAD

**Issue by Kyzegs** [DisnakeDev/disnake#189]
### Summary

The library seems to run into an issue (randomly) where the guild's member count is `None`.

### Reproduction Steps

NA

### Minimal Reproducible Code

NA

### Expected Results

NA

### Actual Results

```
Traceback (most recent call last):
  File "bot.py", line 112, in <module>
    libertas_support.run()
  File "bot.py", line 91, in run
    super().run(config.TOKEN)
  File "/usr/local/lib/python3.8/dist-packages/disnake/client.py", line 834, in run
    return future.result()
  File "/usr/local/lib/python3.8/dist-packages/disnake/client.py", line 813, in runner
    await self.start(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/disnake/client.py", line 777, in start
    await self.connect(reconnect=reconnect)
  File "/usr/local/lib/python3.8/dist-packages/disnake/client.py", line 674, in connect
    await self.ws.poll_event()
  File "/usr/local/lib/python3.8/dist-packages/disnake/gateway.py", line 588, in poll_event
    await self.received_message(msg.data)
  File "/usr/local/lib/python3.8/dist-packages/disnake/gateway.py", line 538, in received_message
    func(data)
  File "/usr/local/lib/python3.8/dist-packages/disnake/state.py", line 1167, in parse_guild_member_add
    guild._member_count += 1
TypeError: unsupported operand type(s) for +=: 'NoneType' and 'int'
```

### Intents

NA

### System Information

- Python v3.8.10-final
- disnake v2.2.0-beta
    - disnake pkg_resources: v2.2.0
- aiohttp v3.7.4.post0
- system info: Linux 5.4.0-73-generic #​82-Ubuntu SMP Wed Apr 14 17:39:42 UTC 2021

### Checklist

- [X] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.
- [X] I have removed my token from display, if visible.

**Comment by shiftinv** [comment]
This was a known issue in the (short-lived) version 2.2.0, it should be fixed in later versions; does this still occur on the latest version?

**Comment by Kyzegs** [comment]
I'm not sure. I'll update my bot instance.

**Comment by Victorsitou** [comment]
@Kyzegs Any news on this issue?

**Comment by Kyzegs** [comment]
As mentioned by @shiftinv, the issue got fixed in 2.2.0.
